docs/dev/cli/cli-commands.md

```diff
@@ -0,0 +1,86 @@
+## Build the CLI from Source
+Steps to build the **TOSCAna Command Line Interface** from sources:
+1. Clone the project.
+
+    `
+    git clone https://github.com/StuPro-TOSCAna/TOSCAna.git
+    `
+
+2. Enter the cli directory.
+
+    `
+    cd TOSCAna/cli
+    `
+
+3. Build the project with:
+
+    `
+    mvn package
+    `
+
+4. You can find the generated jar file `cli-1.0-SNAPSHOT-jar-with-dependencies.jar` in the target folder.
+
+## Usage
+How to use **TOSCAna CLI**:
+
+1. Create an Alias `toscana` on Unix Systems for easier usage:
+
+	`
+    alias toscana='java -jar cli-1.0-SNAPSHOT-jar-with-dependencies.jar'
```

Comment:
This will only work if current working directory does not change. But as a user, I want to call `toscana` in every dir!
Change to:
`alias toscana="java -jar $(readlink -f cli-1.0-SNAPSHOT-jar-with-dependencies.jar)"`
readlink returns the absolute path of that file.